apps/asap-server/src/handlers/webhooks/webhook-calendar-created.ts

```diff
@@ -0,0 +1,176 @@
+import debug from 'debug';
+import AWS from 'aws-sdk';
+import Joi from '@hapi/joi';
+import { Auth } from 'googleapis';
+import { framework as lambda } from '@asap-hub/services-common';
+import { WebhookPayload, Calendar } from '@asap-hub/squidex';
+import {
+  awsRegion,
+  googleApiCredentialsSecretId,
+  googleApiUrl,
+  asapApiUrl,
+} from '../../config';
+import { http } from '../../utils/instrumented-framework';
+import { Handler } from '../../utils/types';
+import validateRequest from '../../utils/validate-squidex-request';
+import Calendars, { CalendarController } from '../../controllers/calendars';
+
+const logger = debug('asap-server');
+
+export const webhookCalendarCreatedHandlerFactory = (
+  subscribe: SubscribeToEventChanges,
+  unsubscribe: UnsubscribeFromEventChanges,
+  calendarController: CalendarController,
+): Handler =>
+  http(
+    async (request: lambda.Request): Promise<lambda.Response> => {
+      validateRequest(request);
+
+      const bodySchema = Joi.object({
+        type: Joi.string().required(),
+        payload: Joi.object({
+          id: Joi.string().required(),
+          data: Joi.object().required(),
+          dataOld: Joi.object(),
+        })
+          .unknown()
+          .required(),
+      })
+        .unknown()
+        .required();
+
+      const { payload, type: event } = lambda.validate(
+        'body',
+        request.payload,
+        bodySchema,
+      ) as WebhookPayload<Calendar>;
+
+      if (event === 'CalendarsUpdated') {
+        if (
+          !payload.dataOld ||
+          !payload.dataOld.id ||
+          payload.dataOld.id.iv === payload.data.id.iv
+        ) {
+          return {
+            statusCode: 200,
+            payload: payload.data.id,
+          };
+        }
+
+        if (payload.dataOld.resourceId) {
+          try {
+            await unsubscribe(payload.dataOld.resourceId?.iv, payload.id);
+          } catch (error) {
+            logger(error);
```

Comment:
Sorry to be picky, but can we add some context to the log messages so that we don't just get random errors or JSONs on the log streams?